packages/frontend/src/components/modules/documents/SideMenu/HeadBar/HeadBarTitle.tsx

```diff
@@ -0,0 +1,30 @@
+import { useRouter } from 'next/dist/client/router'
+import { useEffect, useState } from 'react'
+import classNames from 'classnames'
+
+import style from './HeadBar.module.scss'
+import { IPropsMenu } from '../type/IPropsMenu'
+
+const HeadBarTitle = ({ btnOnClick, isOpen, data }: {btnOnClick: () => void, isOpen: boolean, data: IPropsMenu[]}): JSX.Element => {
+  const [titleText, setTitleText] = useState('')
+  const router = useRouter()
+  useEffect(() => {
+    data.forEach((dats) => {
+      Object.entries([dats]).map((item) => {
+        if (router.pathname === item[1].href) {
+          return setTitleText(item[1].title)
+        }
+        return 'Нет такой страницы'
+      })
```

Comment:
Почему не сделать сразу вот так (у тебя же уже есть массив с объектами)
```suggestion
   data.map((item) => {
      if (router.pathname === item.href) {
        return setTitleText(item.title)
      }
      return 'Нет такой страницы'
    })
```

Reply:
Теперь по поводу местоположения данной логики.
Ее нужно вынести в родитель HeadBar, т.к. нужно навешивать 2 активных класса:
1. один на стрелку (то что ты сейчас делаешь)
2. второй на пункт меню (нужно подсветить цветом какой пункт сейчас выбран, см. макет)

Reply:
Но это возможно лучше сделать уже на новой архитектуре, т.к. немного изменится то. как нужно собирать компонент (HeadBar) и как его раскладывать по папочкам.